packages/SketchMorph2-Tests.package/M2DrawingComponentTests.class/instance/drawDiagonalCirclesFrom.to..st

```diff
@@ -6,4 +6,4 @@ drawDiagonalCirclesFrom: aStartNumber to: anEndNumber
 			userActionMoveDrawing: ((4 * x) + 2 ) asPoint;
```

Comment:
I don't think there should be a space in front of the closing parenthesis.

Reply:
As above.